Assets/Scripts/Inventory/Item.cs

```diff
@@ -2,18 +2,26 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-// Base class to hold all types of items
-public class Item : MonoBehaviour
+namespace FunMath
 {
-    public enum ItemType
+    // Base class to hold all types of items
+    [System.Serializable]
+    public class Item : MonoBehaviour
     {
-        // These are the type of items you can pick up, aka our math 'operators'
-        Addition,
-        Subtraction,
-        Multiply,
-        Divide,
-        None
+        public enum ItemType
+        {
+            // These are the type of items you can pick up, aka our math 'operators'
+            Addition = 0,
+            Subtraction,
+            Multiply,
+            Divide,
+
+            OperationMax,
+
+            Number,

```

Comment:
I'm not sure how I feel about the item type being applied to both numbers and operators as they seem like very different types of items.
But, if you've got it working and it isn't causing you issues, we can go with it! 😃

Reply:
I agree with Logan. I think we should have two lists. One stores all the math operators, and the other stores all the numbers. This could be easier to get the operator and number separately in HUD UI.

Reply:
You'd just have to have a typed Inventory using Generics like `Inventory<Operator>` and `Inventory<Modifier>`.  That way, you can accept items with a typed function like `AddItem<T>(Item<T> item, int count)` or the like.  So I guess your item would need to know what type it will need to support and the inventory will need to know what type that is too, and as long as they agree, you're good!  That way we don't accidentally start mixing item types in the same inventory.

Reply:
Generics are a good idea, but keep in mind we'd lose a bunch of debugging capabilites via the inspector because Unity can't figure out how generics can be reflected. I'd need to test it, last I checked they don't support this, which makes things annoying.